e2e-tests/cypress/tests/integration/channels/enterprise/system_console/channel_moderation/post_reactions_spec.ts

```diff
@@ -43,15 +46,15 @@ describe('MM-23102 - Channel Moderation - Post Reactions', () => {
             testTeam = team;
             testChannel = channel;
 
-            cy.apiCreateGuestUser().then(({guest}) => {
+            cy.apiCreateGuestUser({}).then(({guest}) => {
                 guestUser = guest;
 
                 cy.apiAddUserToTeam(testTeam.id, guestUser.id).then(() => {
                     cy.apiAddUserToChannel(testChannel.id, guestUser.id);
                 });
 
                 // Post a few messages in the channel
-                visitChannel(admin, testChannel, testTeam);
+                visitChannel(admin as UserProfile, testChannel, testTeam);
```

Comment:
So now the casting is moved out a bit, can we just have `getAdminAccount` return a `UserProfile`?

Reply:
Yeah, sure.